LayoutTests/TestExpectations

```diff
@@ -3128,13 +3128,13 @@ webkit.org/b/172609 fetch/closing-while-fetching-blob.html [ Skip ]
 
 webkit.org/b/175291 imported/w3c/web-platform-tests/css/css-ui/outline-018.html [ Skip ]
 
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-014.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-015.html [ ImageOnlyFailure ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-014.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-015.html [ Pass ]
 webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-016.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-018.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-019.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-024.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-025.html [ ImageOnlyFailure ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-018.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-019.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-024.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-025.html [ Pass ]
```

Comment:
Removing these lines would be better than marking them as `Pass`

Reply:
done